### ISSUE TITLE
Add Upload Button on Video Block

### ISSUE THREAD

**Issue by monika-12** [WordPress/gutenberg#4426]
![image-block](https://user-images.githubusercontent.com/17018981/34873173-1649f2da-f7ba-11e7-99cc-fc3ef656d7de.png)
![video-block](https://user-images.githubusercontent.com/17018981/34873174-167b48b2-f7ba-11e7-8579-fb3535898b1b.png)

**Issue Overview**
It is the enhancement. Add Upload Button on Video block like Image block.

 **Screenshots / Video**

![video-block](https://user-images.githubusercontent.com/17018981/34873203-25bda824-f7ba-11e7-9c3b-3e09b8306eb5.png)

![image-block](https://user-images.githubusercontent.com/17018981/34873146-fc98b402-f7b9-11e7-8f87-c8b04974ac39.png)

**Comment by Soean** [comment]
If we add an upload button, then also to the Audio Block

Screenshot from Master:
![media](https://user-images.githubusercontent.com/695201/34873654-3b6d081e-f796-11e7-97fe-26f99b5e4059.png)

**Comment by monika-12** [comment]
Yes @Soean Either we can add upload button like Image Block or Add URL feature like Audio.

**Comment by monika-12** [comment]
@Soean  URL option on video block not displayed at my end. I am using Google Chrome on Windows 10. It's displaying only "Add from Media Library" Button like I provided in screenshot.

**Comment by Soean** [comment]
@monika-12 This is the master branch of the Github repository. The URL option was added a few day ago in #4196. You will see it, when Gutenberg 2.0 is released today.

**Comment by monika-12** [comment]
Okay @Soean got it. Thanks
